client/asset/eth/multirpc_live_test.go

```diff
@@ -56,55 +56,25 @@ func TestRPC(t *testing.T) {
 }
 
 var freeServers = []string{
-	// https://www.alchemy.com/chain-connect/chain/ethereum
-	// Passing 03-26-2024
-	"https://rpc.builder0x69.io",                                          // Limits unknown
-	"https://eth.drpc.org",                                                // 210 million Compute Units (CU) per 30-day period - 20 CU/req
-	"https://rpc.ankr.com/eth",                                            // 30 req per second, no WebSockets (premium-only)
-	"https://ethereum.blockpi.network/v1/rpc/public",                      // 10 req per sec, no WebSockets (premium-only)
-	"https://rpc.flashbots.net",                                           // Limits unknown
-	"wss://eth.llamarpc.com",                                              // Limits unknown
-	"https://eth-mainnet.nodereal.io/v1/1659dfb40aa24bbb8153a677b98064d7", // Limits might be 100M compute units at 300 CU/s
-	// Failing 03-26-2024
-	"https://eth-mainnet.gateway.pokt.network/v1/5f3453978e354ab992c4da79", // connect error: failed to connect to even a single provider among: pokt.network
-	"https://ethereum.publicnode.com",                                      // "TransactionReceipt" error: not found
-	"https://nodes.mewapi.io/rpc/eth",                                      // connect error: failed to connect to even a single provider among: mewapi.io
-	"https://eth.api.onfinality.io/public",                                 // connect error: failed to connect to even a single provider among: onfinality.io
-	"https://eth-mainnet-public.unifra.io",                                 // connect error: failed to connect to even a single provider among: unifra.io
-	"https://cloudflare-eth.com/",                                          // "SuggestGasTipCap" error: Method not found
```

Comment:
I think we should leave the failing ones in, even if you want to comment out so we know what didnt work in the past.